lib/octokit/client/licenses.rb

```diff
@@ -1,43 +1,35 @@
+# frozen_string_literal: true
+
 module Octokit
   class Client
-
-    # Methods for licenses API
+    # Methods for the Licenses API
     #
+    # @see https://developer.github.com/v3/licenses/
     module Licenses
-
-      # List all licenses
+      # List commonly used licenses
+      #
       #
-      # @see https://developer.github.com/v3/licenses/#list-all-licenses
       # @return [Array<Sawyer::Resource>] A list of licenses
-      # @example
-      #   Octokit.licenses
-      def licenses(options = {})
-        options = ensure_api_media_type(:licenses, options)
-        paginate "licenses", options
+      # @see https://developer.github.com/v3/licenses/#list-commonly-used-licenses
+      def commonly_used(options = {})
+        get 'licenses', options
       end
 
-      # List an individual license
+      # Get an individual license
       #
+      # @param license [String] The license of the license
```

Comment:
`The license of the license` isn't a great description but I feel like a better fix than changing our generator would be to rename the parameter in the openapi spec to `license name parameter`. I _think_ this is the preferred style in our openapi guidelines and definitely more consistently with the parameters of other endpoints. But it also seems to take the key of the license which is different from the name of the license as returned by the api, so maybe we should suggest the `license key parameter`? 🤔